fetch.bs

```diff
@@ -4363,6 +4363,28 @@ method, when invoked, must run these steps:
 <p>The <a>value pairs to iterate over</a> are the return value of running
 <a for="header list">sort and combine</a> with the <a for=Headers>header list</a>.
 
+<h3 id=fetch-signal>Fetch signal</h3>
+
+<pre class=idl>
+[Exposed=(Window,Worker)]
+interface FetchSignal : CancelationSignal {
+  // Nothing additional exposed here yet
+};
+</pre>
+
+<h3 id=fetch-controller>Fetch controller</h3>
+
+<pre class=idl>
+[Constructor(), Exposed=(Window,Worker)]
+interface FetchController : CancelationController {
+  [SameObject] readonly attribute FetchSignal signal;
+  // Nothing additional exposed here yet
+};
+</pre>
```

Comment:
Well, if you pass in the superclass or a differently subclass, presumably we'd reflect that instance directly on the constructed `Request` object and not some copy. And therefore that would have to specify the superclass.